### ISSUE TITLE
Wide text despite pp_margin

### ISSUE THREAD

**Issue by cpitclaudel** [ejgallego/coq-serapi#164]
(This is probably a doc request rather than a bug)

I'm trying to print within a given width; I tried the following:

```
("query37"("Print"(("pp_format""PpStr")("pp_depth""25")("pp_margin""49"))("CoqExpr"(("v"("CProdN"(("CLocalAssum"((("v"("Name"("Id""n'0")))("loc"())))("Default""Explicit")(("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""nat")))("loc"()))()))("loc"()))))(("v"("CNotation"("InConstrEntrySomeLevel""_ -> _")(((("v"("CNotation"("InConstrEntrySomeLevel""_ < _")(((("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""n'0")))("loc"()))()))("loc"()))(("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""n")))("loc"()))()))("loc"())))()()())))("loc"()))(("v"("CIf"(("v"("CApp"(()(("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""check_even")))("loc"()))()))("loc"())))(((("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""n'0")))("loc"()))()))("loc"()))()))))("loc"()))(()())(("v"("CApp"(()(("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""isEven")))("loc"()))()))("loc"())))(((("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""n'0")))("loc"()))()))("loc"()))()))))("loc"()))(("v"("CNotation"("InConstrEntrySomeLevel""~ _")(((("v"("CApp"(()(("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""isEven")))("loc"()))()))("loc"())))(((("v"("CRef"(("v"("Ser_Qualid"("DirPath"())("Id""n'0")))("loc"()))()))("loc"()))()))))("loc"())))()()())))("loc"()))))("loc"())))()()())))("loc"()))))("loc"())))))
(Answer query37 Ack)
(Answer query37(ObjList((CoqString"forall n'0 : nat,\nn'0 < n -> if check_even n'0 then isEven n'0 else ~ isEven n'0"))))
(Answer query37 Completed)
```

I'm surprised that despite `pp_margin` being set to `49`, the resulting `CoqString` seems to have `n'0 < n -> if check_even n'0 then isEven n'0 else ~ isEven n'0`, which is 62-characters long.

What am I missing?
Thanks a lot :)

**Comment by ejgallego** [comment]
I can't reproduce [on the v8.10 branch]:
```
(Answer query37(ObjList((CoqString"forall n'0 : nat,\nn'0 < n ->\nif check_even n'0\nthen isEven n'0\nelse ~ isEven n'0"))))
```

**Comment by cpitclaudel** [comment]
Ah, great news! I'm on `8.9.0+0.6.1`

**Comment by ejgallego** [comment]
I was thinking of doing a 0.6.2 release but IMHO this alone doesn't justify it, there is nothing else on the queue. Please reopen if you wanna get this fixed for 0.6 / Coq 8.9
